packages/gatsby-theme-apollo-docs/src/components/page-content.js

```diff
@@ -221,6 +248,11 @@ export default function PageContent(props) {
             imagesLoaded={imagesLoaded === imagesToLoad}
           />
         )}
+        { props.ffWidgetId && (
+          <FeedbackLink>
+            <IconStar style={starIconOffset} /> Rate article
```

Comment:
You could put this line inside of the `FeedbackLink` component, and only render `<FeedbackLink />` here